src/viem/index.ts

```diff
@@ -1,9 +1,10 @@
 import networkConfig, { Network } from 'src/web3/networkConfig'
 import { createPublicClient, http } from 'viem'
+import { celo, celoAlfajores } from 'viem/chains'
 
 export const publicClient = {
   [Network.celo]: createPublicClient({
-    chain: networkConfig.viemChain.celo,
+    chain: networkConfig.viemChain.celo as typeof celo | typeof celoAlfajores, // this type allows us to use feeCurrency on publicClient methods
```

Comment:
can we just use this type on networkConfig (even if you have to cast there) instead of casting in multiple places?

Reply:
The type on networkConfig is `ViemChain` so it can allow any viem chain. So I am not sure how we can specify the exact type there. Did you have anything specific in mind?

Reply:
...not really lol